doc/RELEASE_WALKTHROUGH.rst

```diff
@@ -122,7 +122,7 @@ repository::
 
 Sanity check::
 
-    $ python3 runtests.py -m "full"
+    $ spin test -m full
```

Comment:
I'd like to keep using the module form "python3 -m spin test -m full". It is needed when building with different versions of Python, which can be useful for testing.

Reply:
That probably isn't useful, because `spin` will not be installed globally (unlike `pip` often, and `pytest` sometimes) but only in a development environment (venv, conda env). So it'll either be missing, or automatically find the correct python interpreter. Using the `python3 -m` form doubles the number of characters to type without much of an upside. You can do it if you want, but it doesn't seem good to recommend in the docs.

Reply:
`sudo apt install spin` works, and will result in very much the wrong thing happening since that `spin` is something else altogether.

Reply:
That should be obvious then I'd think. 

If this comment was specific to the release walkthrough, then please ignore my comment and go with whatever @charris prefers. I interpreted the comment as applying to everything.